dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/JSONFieldsTest.java

```diff
@@ -75,21 +85,41 @@ public void cleanUp() throws Exception {
      */
     @Test
     public void testMessageFieldNamesEnv() throws Exception {
-        setUp(server_env);
-        List<String> lines = server_env.findStringsInFileInLibertyServerRoot("log3", MESSAGE_LOG);
+        setUp(server_env_rename);
+        List<String> lines = server_env_rename.findStringsInFileInLibertyServerRoot("log3", MESSAGE_LOG);
         assertTrue("The message field name was not formatted in the new configuration in messages.log.", lines.size() > 0);
     }
 
+    /*
+     * This test sets the "WLP_LOGGING_JSON_FIELDS" attribute in the server.env and verifies the omission in the messages.log file.
+     */
+    @Test
+    public void testMessageFieldNamesOmitEnv() throws Exception {
+        setUp(server_env_omit);
+        List<String> lines = server_env_omit.findStringsInFileInLibertyServerRoot("\"message\"", MESSAGE_LOG);
+        assertTrue("The message field name was omitted in messages.log.", lines.size() > 0);
```

Comment:
lines.size() == 0 (since you're omitting) maybe change to omit "loglevel" field instead